bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/ComponentUtils.java

```diff
@@ -165,4 +166,20 @@ public static ContentPolicy getPolicy(String contentPath, ResourceResolver resou
         return policy;
     }
 
+    /**
+     * Takes in a single variable and returns an array of size 1 consisting of the same variable.
+     * Useful to directly iterate on, when something returns either a single value or an array of values of a particular type.
+     *
+     * @param param of any type T
+     * @return array of size 1 having the same variable
+     */
+    public static <T> T[] convertToArray(T param) {
```

Comment:
Why is this change required, this is not used anywhere, please revert unrequired changes and raise a PR